cliqz.cfg

```diff
@@ -18,6 +21,25 @@ let extensionUrls = {
 
 let prefs = Cc["@mozilla.org/preferences-service;1"].getService(Ci.nsIPrefBranch);
 
+function verifyAddons() {
+  Cu.import("resource://gre/modules/addons/XPIProvider.jsm");
+  let versionChecker = Cc["@mozilla.org/xpcom/version-comparator;1"]
+                         .getService(Components.interfaces.nsIVersionComparator);
+  let distributionVersion = prefs.getCharPref("distribution.version");
+
+  let lastDistributionVersion;
+  try {
+    lastDistributionVersion = prefs.getCharPref("distribution.lastVersion");
+  } catch(e) {
+    lastDistributionVersion = 0;
+  }
+
+  if (versionChecker.compare(distributionVersion, lastDistributionVersion) > 0) {
+    XPIProvider.verifySignatures();
+    prefs.setCharPref("distribution.lastVersion", distributionVersion);
```

Comment:
There is "extensions.lastPlatformVersion" which serves the same purpose and is already maintained by `AddonManager`.
And actually I think it's best to do it where all other similar job is done – in `AddonManagerInternal.startup`